cni/pkg/plugin/plugin.go

```diff
@@ -278,7 +278,7 @@ func doRun(args *skel.CmdArgs, conf *Config) error {
 	}
 
 	if !pi.Containers.Contains(ISTIOPROXY) {
-		log.Infof("excluded because it does not have istio-proxy container (have %v)", sets.SortedList(pi.Containers))
+		log.Infof("excluded because it does not have istio-proxy container (have %s)", sets.SortedList(pi.Containers))
```

Comment:
a slice is not `%s` compatible?

Reply:
You are right, this is a []T not Set[T], fixed